packages/mui-joy/src/Button/Button.tsx

```diff
@@ -98,10 +103,11 @@ const ButtonRoot = styled('button', {
       // TODO: discuss the transition approach in a separate PR. This value is copied from mui-material Button.
       transition:
         'background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms, color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms',
-      ...theme.typography.body1,
+      fontFamily: theme.vars.fontFamily.body,
+      fontSize: theme.vars.fontSize.md,
+      lineHeight: 1,
```

Comment:
Just double checking if this is intended. Shouldn't it be `1rem`?

Reply:
line-height: 1, means that it is equal to the font-size. I think it is better in this case because the button font-size can vary based on the `size` prop. The reason I use line-height 1 is that I can't find a better number otherwise you will see that the text is not aligned center inside the button. It is quite the same issue as in https://github.com/mui/material-ui/pull/30240

Reply:
I see. Thanks for explaining!